README.md

```diff
@@ -702,6 +702,14 @@ const { results } = await Calculator.calculateMeasureReports(measureBundle, pati
 
 **NOTE**: Measure logic highlighting is not available when calculating a population `MeasureReport` with `reportType = 'summary'`
 
+However, highlighted HTML provided by `fqm-execution` is not solely based on whether individual pieces of the measure logic CQL had "truthy" or "falsy" values. When there are populations that depend on each other, a relevance hierarchy is built to ensure that populations that are subsets of others aren't considered for membership until membership is determined for their superset population. If a statement's superset statement is false, then no highlighting will appear, regardless of whether that statement's value is "truthy" or "falsy". This pattern reflects the actual population results that are being returned during calculation.
+
+The following example shows what logic highlighting would look like for a proportion boolean measure where the Initial Population evaluates to false but the Denominator and Numerator statements both evaluated to true.
```

Comment:
Not entirely pleased with where I decided to add this so let me know if you think there's a better spot

Reply:
Yeah, I think "This process is similar to proportion measure scoring for proportion measures depicted in [this flowchart](https://build.fhir.org/ig/HL7/cqf-measures/measure-conformance.html#proportion-measure-scoring)." is a little awkward. 

I would maybe just put it after your last statement since it only applies to the proportion measure example. And can just mention that the relevance calculation matches the population semantics for proportion measures defined in the flowchart